ricecooker/classes/nodes.py

```diff
@@ -150,7 +153,7 @@ def validate(self):
             assert isinstance(self.thumbnail, str) or self.thumbnail is None
             return super(Channel, self).validate()
         except AssertionError as ae:
-            raise InvalidNodeException("Invalid node: {0} - {1}".format(self.title, ae))
+            raise InvalidNodeException("Invalid channel ({}): {} - {}".format(ae.args[0], self.title, self.__dict__))
```

Comment:
Added the assert error message to these to provide more information (see https://github.com/learningequality/ricecooker/issues/26)